src/views/partials/header.hbs

```diff
@@ -2,8 +2,13 @@
   <span>Stackmatch</span>
   <nav>
     <ul>
-      {{#if user.session}}<li><a href="/auth/github/logout" type="button" id="js-logout">Logout</a></li>{{/if}}
-      {{#if user.github_avatar_url}} <img src="{{user.github_avatar_url}}" alt="my profile picture" class="nav-avatar"> {{/if}}
+      {{#if user.session}}
```

Comment:
Inside `/src/controllers/home.js` the res.render() function does not include the `user` object. If it is included it will fix part of #84